src/apps/LandingRegister/LandingRegister.js

```diff
@@ -57,10 +56,9 @@ export default Radium(React.createClass({
     };
   },
 
-  handleButtonPress() {
-    this.setState({
-      registerState: true
-    });
+  submitForm(model) {
+    console.log('Model: ', model);
```

Comment:
As far as I remember we are using in Syncano console.debug, which can be easily cleaned during production build.